models/FRA_coefs.csv

```diff
@@ -1,7 +1,17 @@
 adm0,policy,suffix,beta,se
+<<<<<<< Updated upstream

```

Comment:
@jeanettelt @sannanphan I know this already got merged but just a flag it looks like we have some merge conflicts in a few of the files that got merged in with this PR. I'm not sure which one is "right" but could one of you take a look when you get a chance and resolve? It looks like `FRA_coefs.csv`, `FRA_preds.csv`, and  `FRA_reg_data.csv`